### ISSUE TITLE
Project idea: Full support for DBI in perl6

### ISSUE THREAD

**Issue by Tux** [perl-gsoc-2019/ideas#30]
Plus basic support for
• DBD::Pg
Plus experimental support for
• DBD::File
• DBD::MariaDB
And maybe
• DBD::Oracle

**Comment by patzim** [comment]
Are you able and willing to mentor for this?

**Comment by Tux** [comment]
> Are you able and willing to mentor for this?

Not really. No time and not enough experience in perl6 to make me a good mentor
